aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+|-----|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| US1 | Usage Instructions | Describe how to use algorithm: data pre-processing guidelines, training data format (e.g, mandatory fields), minimum data row requirements, and guidelines to create good model (e.g., identify important hyperparameters and values). <br><br>Also, add details of approximate duration of training based on data. Clarify any feature engineering (e.g., scaling, imputing of values) performed by the algorithm. | Mandatory                    | The algorithm is a X pass algorithm and a training on XXX.XXXX instance for a dataset of size XX MB takes XX minutes of compute.You are recommended to provide 50 rows required in the training data.<br>For large datasets with > 100,000 rows or > 1,000 columns, use larger instance type. <br><br>For best results: provide all your data to AutoGluon as train_data rather than splitting a validation set yourself, specify which eval_metric will be used to evaluate predictions.<br> <br>The algorithm handles missing data, manual feature transformations, data splitting, model selection, algorithm selection, hyperparameter selection and tuning, ensembling multiple models, and repeating the process when data changes.         |
```

Comment:
```suggestion
| US1 | Usage Instructions | Describe how to use algorithm: data pre-processing guidelines, training data format (e.g, mandatory fields), minimum data row requirements, and guidelines to create good model (e.g., identify important hyperparameters and values). <br><br>Also, add details of approximate duration of training based on data. Clarify any feature engineering (e.g., scaling, imputing of values) performed by the algorithm. | Mandatory                    | The algorithm is a X pass algorithm and a training on XXX.XXXX instance for a dataset of size XX MB takes XX minutes of compute.You are recommended to provide 50 rows required in the training data.<br>For large datasets with > 100,000 rows or > 1,000 columns, use larger instance type. <br><br>For best results, provide all your data to AutoGluon as train_data rather than splitting a validation set yourself and specify which eval_metric will be used to evaluate predictions.<br><br>The algorithm handles missing data, manual feature transformations, data splitting, model selection, algorithm selection, hyperparameter selection and tuning, ensembling multiple models, and repeating the process when data changes.         |
```